### ISSUE TITLE
Windows Platfom Progress

### ISSUE THREAD

**Issue by ravensorb** [zumero/DataGrid#6]
Is there enough implemented in the Windows Platform for this to be usable?

**Comment by jeremysheeley** [comment]
Yes, Windows is usable. One caveat is that there is an issue with Xamarin.Forms 2.0.0 that will cause errors on Windows platforms. You can follow the progress of that fix at:

https://bugzilla.xamarin.com/show_bug.cgi?id=36328

**Comment by jeremysheeley** [comment]
Also, to see source for a windows project using DataGrid, you can download the Zumero for SQL Server Test App at http://zumero.com/dev-center/zss/

**Comment by ravensorb** [comment]
Excellent.  One other question, is there a way to bind the "columns" to a collection?  I am looking for something that has most of the functionality of a data grid that has 1 row and scrolls horizontally.

**Comment by ravensorb** [comment]
Cool thanks

**Comment by jeremysheeley** [comment]
It's not a thing that we've tried, but it should be possible. You may end up with memory issues if you're using lots of column headers, as they have a different lifecycle than grid cells.

**Comment by balaameex** [comment]
Is it support for nested Grid view? In one of my project has the requirement of nested gridview. I tried with default List view control it is not working with Android and crashes the app.while scrolling. So that I want to confirm It will work for Nested Grid View.

**Comment by ericsink** [comment]
@balaameex  In general, I suspect that anytime you nest a scrolling thing inside another scrolling thing, it will not work well.  I don't know exactly what will happen if you try to nest DataGrids, but I don't think it will work.
